src/main/java/dukeexception/InvalidDateTimeException.java

```diff
@@ -0,0 +1,14 @@
+package dukeexception;
+
+public class InvalidDateTimeException extends DukeException {
+
+    /**
+     * Return message of the Thrown InvalidDateTimeException.
+     *
+     * @return String to be printed.
+     */
+    @Override
+    public String getMessage() {
+        return "ERROR: Date/time format is wrong!";
```

Comment:
Perhaps consider telling the user the expected correct input, so that they can try again?